models/hugging_face/bert-base-cased/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  type: azureblob
+  storage_name: automlcesdkdataresources
+  container_name: models
+  container_path: HuggingFace/bert-base-cased/5532cc5/mlflow_model_folder
```

Comment:
>HuggingFace/bert-base-cased/5532cc5/mlflow_model_folder [](http://example.com/codeflow?start=18&length=55)

I believe we are waiting on an update to the model related to local inference. Let us wait for that to be available before merging.

Reply:
Yes, waiting for same